Application/src/Parser.java

```diff
@@ -0,0 +1,99 @@
+import Model.diagramObject;
+import com.google.gson.Gson;
+
+/**
+ * @author Rashad Kamsheh & Isabelle Törnqvist
+ * @since 2017-10-16
+ */
+
+public class Parser {
+
+    // odd number for the First Diagram counter
```

Comment:
Does not explain the WHY so the comment doesn't provide anything the code doesn't.

Reply:
Same applies to the next comment